packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -286,15 +286,23 @@ export default class QuanticCategoryFacet extends LightningElement {
   }
 
   get values() {
-    return this.state?.values ?? [];
+    if (!this.state?.valuesAsTrees?.length) {
+      return [];
+    }
+
+    if (this.state?.selectedValueAncestry?.length > 0) {
+      return this.state?.activeValue?.children ?? [];
+    }
+
+    return this.state?.valuesAsTrees;
```

Comment:
Can this ever be something that isn't an array?

If we return something that isn't an array here, for example a single value maybe? The component will crash.
It's because we basically make a `forEach` on the returned value of this function.